packages/caliper-core/lib/worker/caliper-worker.js

```diff
@@ -39,6 +39,7 @@ class CaliperWorker {
         this.connector = connector;
         this.workerIndex = workerIndex;
         this.messenger = messenger;
+        this.rateControlClass = RateControl;
```

Comment:
We don't need this change.

Reply:
i added this so i can manipulate the rate control in the test cases, what other way would you suggest i achieve this?

Reply:
Have a look a the RecordRate test. It shows a way to manipulate an existing rate control with your own mock one